x/concentrated-liquidity/fees_test.go

```diff
@@ -55,59 +56,83 @@ func (s *KeeperTestSuite) TestInitializeFeeAccumulatorPosition() {
 			DefaultLowerTick,
 			DefaultUpperTick,
 			DefaultPositionId,
+			DefaultLiquidityAmt,
 		}
 	)
 
-	withOwner := func(posId positionFields, owner sdk.AccAddress) positionFields {
-		posId.owner = owner
-		return posId
+	withOwner := func(posFields positionFields, owner sdk.AccAddress) positionFields {
+		posFields.owner = owner
+		return posFields
 	}
 
-	withUpperTick := func(posId positionFields, upperTick int64) positionFields {
-		posId.upperTick = upperTick
-		return posId
+	withUpperTick := func(posFields positionFields, upperTick int64) positionFields {
+		posFields.upperTick = upperTick
+		return posFields
 	}
 
-	withLowerTick := func(posId positionFields, lowerTick int64) positionFields {
-		posId.lowerTick = lowerTick
-		return posId
+	withLowerTick := func(posFields positionFields, lowerTick int64) positionFields {
+		posFields.lowerTick = lowerTick
+		return posFields
 	}
 
-	withPositionId := func(posId positionFields, positionId uint64) positionFields {
-		posId.positionId = positionId
-		return posId
+	withPositionId := func(posFields positionFields, positionId uint64) positionFields {
+		posFields.positionId = positionId
+		return posFields
+	}
+
+	withLiquidity := func(posFields positionFields, liquidity sdk.Dec) positionFields {
+		posFields.liquidity = liquidity
+		return posFields
 	}
 
 	clKeeper := s.App.ConcentratedLiquidityKeeper
 
 	type initFeeAccumTest struct {
-		name           string
-		positionFields positionFields
+		name             string
+		positionFields   positionFields
+		isPositionUpdate bool
```

Comment:
Might be missing it but where is this test field currently used?

Reply:
No, removed this obsoleted flag